### ISSUE TITLE
Show an error message if an import template has invalid columns

### ISSUE THREAD

**Issue by BarnaBartha** [SORMAS-Foundation/SORMAS-Project#3530]
# Steps to Reproduce
1. Go to cases > Import
2. Select a csv file that is not the cases import template
3. Start data import
4. Results in import successful with 0 imported 0 errors 0 processed etc..

# Expected result
Import should fail with useful information to user about the template not being the correct one.

**Comment by vidi42** [comment]
I think that because of https://github.com/hzi-braunschweig/SORMAS-Project/issues/3453 the Export is more similar to the Import Template now so it can be used to do an import.

However I could not reproduce the behavior you described on my local machine or https://test.sormas.netzlink.com.

Tried the following:
1. with an admin, go to cases and try to import an export of contacts -> resulted in an Error Report
2. with an admin, go to cases and try to import a basic export as line listing -> resulted in an Error Report
3. with an admin, go to cases and try to import an empty csv -> import failed with no Error Report

@BarnaBartha can you provide more details on how to reproduce this and where?

**Comment by leventegal-she** [comment]
Couldn't reproduce. With an arbitrary not SORMAS related csv the result is `341/341 Processed 0 Imported | 341 Error(s) | 0 Skipped as Duplicates | 0 Skipped`.

**Comment by BarnaBartha** [comment]
@vidi42 @leventegal-she I cannot reproduce it any more either. We can close this as it is obsolete and probably was fixed in the meantime.

**Comment by leventegal-she** [comment]
@BarnaBartha the importExample.csv you provided has only two processable rows; for case import these rows should contain class names and field names. This means there is nothing to import so the result "0 imported 0 errors 0 processed etc.." is correct.

I wouldn't consider not reporting an error that the columns are not ok to be a bug. It's not implemented yet. 
Implementing column validation should be the scope of a feature request.

**Comment by vidi42** [comment]
I agree with @leventegal-she's remark that this looks more like an extension to the current import system.

@markusmann-vg @MateStrysewske do you agree?

**Comment by MateStrysewske** [comment]
Yes, agreed
